### ISSUE TITLE
Ledger Wallet not working with version 0.2.2

### ISSUE THREAD

**Issue by jlam1987** [CityOfZion/neon-wallet#893]
Ledger wallet not working. I am getting the following error message:
"Looking for USB Devices. Please plugin your device and login.
cannot open device with path /dev/hidraw0"

**Comment by snowypowers** [comment]
Hi, can you provide more information?

1. What is the last working version of NEON for you?
2. Have you tried plugging/unplugging the cable? Changing usb port?
3. Is your ledger Nano S running on the latest firmware with the latest NEO app version? Please provide the version numbers.

**Comment by jlam1987** [comment]
Hi there,

1. The last one I've used is 0.1.4 ("Neon_0.1.4_amd64.Linux"), which worked fine. If it helps, I am running Ubuntu 16.04.4 LTS on a Toshiba Chromebook 2 2015 using Crouton.

2. I have tried plugging and unplugging the cable, changing the USB port, and restarting my laptop. I have also managed to use the wallet for other currencies such as ethereum through MyEtherWallet in my Chrome browser and Ark through the official Ark desktop client.

3. My Nano S is on the latest firmware, Secure Element 1.4.1 and MCU 1.5. The NEO app version is 1.1.2.

Thanks for your help.
